config.json

```diff
@@ -1028,24 +1028,24 @@
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-crashlytics",
-        "version": "19.0.1",
-        "nugetVersion": "119.0.1",
+        "version": "19.0.3",
+        "nugetVersion": "19.0.3",
```

Comment:
119

Reply:
I think it is update script, but not in all cases. I fixed 10+ of such errors. I will check next time.
Only GPS. AX seems to be OK. OK there is no hundredts prefix.